src/VideoIntelligence/V1beta1/VideoIntelligenceServiceClient.php

```diff
@@ -0,0 +1,428 @@
+<?php
+/*
+ * Copyright 2017, Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * This file was generated from the file
+ * https://github.com/google/googleapis/blob/master/google/cloud/videointelligence/v1beta1/video_intelligence.proto
+ * and updates to that file get reflected here through a refresh process.
+ *
+ * EXPERIMENTAL: this client library class has not yet been declared beta. This class may change
+ * more frequently than those which have been declared beta or 1.0, including changes which break
+ * backwards compatibility.
+ */
+
+namespace Google\Cloud\VideoIntelligence\V1beta1;
+
+use Google\GAX\AgentHeaderDescriptor;
+use Google\GAX\ApiCallable;
+use Google\GAX\CallSettings;
+use Google\GAX\GrpcConstants;
+use Google\GAX\GrpcCredentialsHelper;
+use Google\GAX\LongRunning\OperationsClient;
+use Google\GAX\OperationResponse;
+use google\cloud\videointelligence\v1beta1\AnnotateVideoProgress;
+use google\cloud\videointelligence\v1beta1\AnnotateVideoRequest;
+use google\cloud\videointelligence\v1beta1\AnnotateVideoResponse;
+use google\cloud\videointelligence\v1beta1\Feature;
+use google\cloud\videointelligence\v1beta1\VideoContext;
+use google\cloud\videointelligence\v1beta1\VideoIntelligenceServiceGrpcClient;
+
+/**
+ * Service Description: Service that implements Google Cloud Video Intelligence API.
+ *
+ * EXPERIMENTAL: this client library class has not yet been declared beta. This class may change
+ * more frequently than those which have been declared beta or 1.0, including changes which break
+ * backwards compatibility.
+ *
+ * This class provides the ability to make remote calls to the backing service through method
+ * calls that map to API methods. Sample code to get started:
+ *
+ * ```
+ * try {
+ *     $videoIntelligenceServiceClient = new VideoIntelligenceServiceClient();
+ *     $inputUri = "";
+ *     $features = [];
```

Comment:
No, it won't. This is a cross-language issue with sample generation - all of the languages will currently have a sample that looks like this, with empty inputUri and features. Right now the protos are still private, but once they move into the public googleapis repo I will file an issue to update this sample generation.